sshuttle/client.py

```diff
@@ -224,7 +224,7 @@ def setup():
                 # No env: Talking to `FirewallClient.start`, which has no i18n.
                 e = None
                 break
-            except OSError as e:
+            except OSError:
```

Comment:
What's the purpose for removing `as e` here? It seems that after this changes, `e` is always `None`.

Reply:
Looks like it might be a mistake. The e value is used outside the exception handler, which is a little bit weird. Feel free to supply a pull request that adds it back.